.travis.yml

```diff
@@ -32,6 +32,7 @@ cache:
 install:
   - pip install --upgrade pip
   - pip install tox
+  - pip install netaddr
```

Comment:
This requirements must be installed in the instance itself, this list of packages to install is for the host that runs the role.

But, it's a good one, since the role uses facts like `ansible_default_ipv4`, some extra packages should be installed. I've got a role [core_dependecies](https://galaxy.ansible.com/robertdebock/core_dependencies) that installs all tools required to make these Ansible facts available.

To have [core_dependecies](https://galaxy.ansible.com/robertdebock/core_dependencies) available, add it to these files:
- requirements.yml
- molecule/default/prepare

(I'll update the documentation later.)